modules/eco-node-maintenance-operator-installation-cli.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="installing-maintenance-operator-using-cli_{context}"]
+= Installing the Node Maintenance Operator by using the CLI
+You can use the OpenShift CLI (`oc`) to install the Node Maintenance Operator.
+
+You can install the Node Maintenance Operator in your own namespace or in the `openshift-operators` namespace. 
+
+To install the Operator in your own namespace, follow the steps in the procedure. 
+
+To install the Operator in the `openshift-operators` namespace, skip to step 3 of the procedure because the steps to create a new `Namespace` custom resource (CR) and an `OperatorGroup` CR are not required.
```

Comment:
To install the Operator in the `openshift-operators` namespace, skip to step 3 of the procedure because steps 1 and 2, creating new `Namespace` custom resource (CR), and an `OperatorGroup` CR, are not required.

Reply:
As discussed, retaining the current format